packages/contracts/test/voting/token-voting.ts

```diff
@@ -17,27 +17,27 @@ import {customError, ERRORS} from '../test-utils/custom-error-helper';
 
 const {deployMockContract} = waffle;
 
-describe('ERC20Voting', function () {
+describe('TokenVoting', function () {
```

Comment:
ThIs file should have the tests for NFT for making it safer. We only use erc20Token address in this file

Reply:
This will be a separate task: https://discord.com/channels/672466989217873929/1045348071942410280/1045705111961358356